contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1159,7 +1159,7 @@ SELECT CAST(c.conname as sys.sysname) as name
   , CAST(NULL as integer) as principal_id 
   , CAST(c.connamespace as integer) as schema_id
   , CAST(conrelid as integer) as parent_object_id
-  , CAST('C' as char(2)) as type
+  , CAST('C' as sys.bpchar(2)) as type
```

Comment:
Change to bpchar without COLLATE is inconsistent with other instances of the same type.  I.e. another comment said we would not be changing types here, but just added collation, this one changes type and does not add collation.

Reply:
Thanks for pointing this out. Yes, we are not going to change any data types. @shah-nirmit please make this change.